code/datums/commodity.dm

```diff
@@ -2212,6 +2212,26 @@
 	upperfluc = 100
 	lowerfluc = -20
 
+/datum/commodity/toygun
+	comname = "Toy Gun"
+	comtype = /obj/item/gun/kinetic/foamdartgun
+	desc = "A toy gun filled with foam darts."
```

Comment:
It may not be filled with darts so the description might be misleading.
Could be better to just say it fires foam darts.